.github/workflows/pull.yml

```diff
@@ -541,3 +541,20 @@ jobs:
       docker-image: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-inductor-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-inductor-build.outputs.test-matrix }}
     secrets: inherit
+
+  linux-jammy-xpu-2025_0-py3_9-build:
```

Comment:
For this, you probably want to add a `sync-tag` to ask linter to keep this job and the one from xpu workflow in sync.  See this example from trunk macos build jobs:

* https://github.com/pytorch/pytorch/blob/main/.github/workflows/trunk.yml#L84
* and https://github.com/pytorch/pytorch/blob/main/.github/workflows/mac-mps.yml#L21

Reply:
Done. Thanks @huydhn